docs/csharp/programming-guide/concepts/object-oriented-programming.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Object-Oriented Programming (C#)"
-ms.date: 02/08/2020
```

Comment:
I wouldn't update the ms.date metadata for changes with this limited scope. (Unless you reviewed everything and you're happy that it's up to date.)

Reply:
I'll keep this in mind moving forward. Typically, I do read the entire doc that I'm working on and usually always update the `ms.date` as part of that. It was something that was common practice in the azure-docs repo. This was updated in February, and in reading through it - everything still sounds accurate.